LiskHQ/lisk
# Introduce inert exceptions for testnet - Fixes #1880

### What was the problem?
Some transactions were invalid but saved in the database without being applied to mem_accounts. We needed a new mechanism to handle these transactions (and not normal exceptions) 
### How did I fix it?
Added new exception handling mechanism called "inert exceptions".
### How to test it?
Sync with testnet from genesisBlock, create snapshot and verify mem_account state.
### Review checklist

* The PR solves #1880
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
